x-pack/plugins/ml/public/alerting/config_validator.tsx

```diff
@@ -0,0 +1,157 @@
+
+    const configContainsIssues =
+      isAlertIntervalTooHigh || Object.values(jobIssues).some((v) => v.length > 0);
+
+    if (!configContainsIssues) return null;
+
+    return (
+      <>
+        <EuiSpacer size={'m'} />
+        <EuiCallOut
+          title={
+            <FormattedMessage
+              id="xpack.ml.alertConditionValidation.title"
+              defaultMessage="Alert condition contains the following issues:"
+            />
+          }
+          color="warning"
+          size={'s'}
+        >
+          <ul>
+            {isAlertIntervalTooHigh ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.alertIntervalTooHighMessage"
+                  defaultMessage="The check interval is higher than the result bucket span of {resultBucketSpan}. Be aware of a delay in receiving notifications."
+                  values={{
+                    resultBucketSpan: resultBucketSpanString,
+                  }}
+                />
+              </li>
+            ) : null}
+
+            {jobIssues.failedJobsIds.length > 0 ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.failedJobsMessage"
+                  defaultMessage="{jobIds} {count, plural, one {job is} other {jobs are}} failed"
+                  values={{
+                    count: jobIssues.failedJobsIds.length,
+                    jobIds: jobIssues.failedJobsIds.join(', '),
+                  }}
+                />
+              </li>
+            ) : null}
+
+            {jobIssues.closedJobIds.length > 0 ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.closedJobsMessage"
+                  defaultMessage="{jobIds} {count, plural, one {job is} other {jobs are}} closed"
+                  values={{
+                    count: jobIssues.closedJobIds.length,
+                    jobIds: jobIssues.closedJobIds.join(', '),
+                  }}
+                />
+              </li>
+            ) : null}
+
+            {jobIssues.stoppedDatafeedJobIds.length > 0 ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.stoppedDatafeedJobsMessage"
+                  defaultMessage="{jobIds} {count, plural, one {job has} other {jobs have}} datafeed closed"
+                  values={{
+                    count: jobIssues.stoppedDatafeedJobIds.length,
+                    jobIds: jobIssues.stoppedDatafeedJobIds.join(', '),
+                  }}
+                />
+              </li>
+            ) : null}
+
+            {jobIssues.deletedDatafeedJobIds.length > 0 ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.deletedDatafeedJobsMessage"
+                  defaultMessage="{jobIds} {count, plural, one {job has} other {jobs have}} datafeed deleted"
```

Comment:
Maybe rephrase to `The datafeed has been deleted for job/jobs {jobIds}`?

Reply:
Fixed in https://github.com/elastic/kibana/pull/92377/commits/8113a9ab3468b772a2ff9f794a40f5780525d483

Reply:
I would avoid the present perfect tense here if it's possible. Consider rephrasing it to `The datafeed is deleted for the following job/jobs: {jobIds}.`